frameworks/digital-outcomes-and-specialists/questions/declaration/transparentContracting.yml

```diff
@@ -1,2 +1,2 @@
-question: Will you be transparent about your financial performance, in line with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
+question: Will you be transparent about your financial performance, and comply with your contractual obligations and government guidance on Open Book Contract Management (OBCM)?
```

Comment:
Think we need to find out which is more commonly used before we get rid of it. Hoping Warren can clear up the confusion on this one.